v1/transform-global.yaml

```diff
@@ -0,0 +1,227 @@
+info:
+  version: '1.0.0'
+  title: Global Transform API
+  description: Global Transform API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /html/{from}/to/{to}{/provider}:
```

Comment:
I agree that putting it under `transform` strongly suggests a format transform (due to historical reasons), but it's still a transformation. If we want to communicate strongly that it's about language, then it would probably need to be moved under `translate` or a similar new hierarchy, but I don't think that is the way to go, as language translation for the same content is a form of transformation.